### ISSUE TITLE
Applied filter should be indicated as such

### ISSUE THREAD

**Issue by schima** [bigpanther/shipanther#248]
-Go into orders or shipments. 
-select filter and apply some filter. 

Note there is no indication of filtered result. User may have no idea especially if screen is empty.

**Comment by harsimranmaan** [comment]
Yes filtering requires a lot of work for the ux. The backend supports many ways of filtering objects but I haven’t managed to implement them all as I’m not sure about the UX
